.gitignore

```diff
@@ -9,5 +9,3 @@
 *.meta
 SHA256.txt
 *.dat
-file1.c
```

Comment:
We need to figure out a better way to protect our test files, your branch keeps deleting them and having identical input samples for everyone to use is important.